shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -164,12 +164,14 @@
  *
  * <p><strong>Launch Screen and Splash Screen</strong>
  *
- * <p>{@code FlutterActivity} supports the display of an Android "launch screen" as well as a
- * Flutter-specific "splash screen". The launch screen is displayed while the Android application
- * loads. It is only applicable if {@code FlutterActivity} is the first {@code Activity} displayed
- * upon loading the app. After the launch screen passes, a splash screen is optionally displayed.
- * The splash screen is displayed for as long as it takes Flutter to initialize and render its first
- * frame.
+ * <p>{@code FlutterActivity} supports the display of an Android "launch screen", which is displayed
+ * while the Android application loads. It is only applicable if {@code FlutterActivity} is the
+ * first {@code Activity} displayed upon loading the app.
+ *
+ * <p>Prior to Flutter 2.5, {@code FlutterActivity} supported the display of a Flutter-specific
+ * "splash screen" that would be displayed after the launch screen passes; this has since been
```

Comment:
Nit: Semicolon -> period. Semicolons tend to confuse non-native english speakers.

Reply:
(optional)